src/styles/variables/colors.scss

```diff
@@ -61,7 +83,11 @@ $color-types: (
     }
 
     @if map-has-key($color-types, $color) {
-        @return map-get($color-types, $color);
+        @if $overlay {
+            @return map-get(map-get($color-types, $color), overlay);
+        } @else {
+            @return map-get(map-get($color-types, $color), color);
+        }
```

Comment:
This also needs to be reverted and looks like it's being pulled in accidentally.